galleries/users_explain/axis/index.rst

```diff
@@ -1,6 +1,15 @@
 .. _users_axis:
 
-Controlling and labelling Axis objects
+Controlling and Labelling Axis Objects
 --------------------------------------
 
-Some good material in artist tutorial that should be cribbed from or used here.
+Matplotlib `~.axis.Axis` objects can be controlled and manipulated
+using the `~.artist.Artist` class. These tutorials cover the basics
```

Comment:
Ah okay. That does make more sense. I was still a little confused on the Axis API, so that really help. I'll try to reword it and work on it today or tomorrow (I am currently in my finals week for school so it may be a little delayed). Thank you!

Reply:
I updated the description somewhat, and also added an inheritance diagram (from the Axis API page) to show how it is a type of Artist object. I also added a reference to that page. Let me know if what I wrote is any better and if I should change/add more to it. Thank you!